src/metabase/lib/drill_thru/quick_filter.cljc

```diff
@@ -76,10 +76,10 @@
         {:name   label
          :filter (operator op field-ref value)})
 
-      (lib.types.isa/string? column)
-      (for [[op label] [[:=  "="]
-                        [:!= "≠"]
-                        [:contains "contains"]
+      (and (lib.types.isa/string? column)
+           (or (lib.types.isa/comment? column)
+               (lib.types.isa/description? column)))
+      (for [[op label] [[:contains "contains"]
                         [:does-not-contain "does-not-contain"]]]
```

Comment:
I'm not certain this is correct.

I read the issue as saying that contains/does-not-contain are appearing where they should not. Those are reserved for comment/description fields. But this also removes `:=` and `:!=`  exact matches for such fields, which seems strange to me.

Reply:
The "specification" is not super clear. Having read the namespace's docstring, I decided this was the intended behavior, because the `:=` and `:!=` seem to be mentioned explicitly for other data types. I'm assuming we expect comments and descriptions to be too long to test with `:=` and `:!=` equal.
@bshepherdson, have you checked the namespace doc? How do you interpret it and why?